supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -239,11 +239,21 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
     env = None
   ))
 
-  val resources: Resources = TesTask.makeResources(
+  lazy val resources: Resources = TesTask.makeResources(
     runtimeAttributes,
     preferedWorkflowExecutionIdentity,
     Option(tesPaths.tesTaskRoot)
   )
+
+  lazy val tags: Map[String, Option[String]] = {
+    // In addition to passing through any workflow labels, include relevant workflow ids as tags.
+    val baseTags = jobDescriptor.workflowDescriptor.customLabels.asMap.map { case (k, v) => (k, Option(v)) }
+    baseTags ++ Map(
+      "workflow_id" -> Option(jobDescriptor.workflowDescriptor.id.toString),
+      "root_workflow_id" -> Option(jobDescriptor.workflowDescriptor.rootWorkflowId.toString),
+      "parent_workflow_id" -> jobDescriptor.workflowDescriptor.breadCrumbs.lastOption.map(_.id.toString)
```

Comment:
Could add a
```
val parentWorkflowId = breadCrumbs.lastOption.map(_.id.toString)
```
to `BackendWorkflowDescriptor` and call it here, would better encapsulate the breadcrumb stuff.

Reply:
I like it, will do!